repo/open.go

```diff
@@ -420,11 +549,11 @@ func readAndCacheRepositoryBlobBytes(ctx context.Context, st blob.Storage, cache
 
 	cacheEnabled := formatBytesCachingEnabled(cacheDirectory, validDuration)
 	if cacheEnabled {
-		b, err := readRepositoryBlobBytesFromCache(ctx, cachedFile, validDuration)
+		b, cacheTime, err := readRepositoryBlobBytesFromCache(ctx, cachedFile, validDuration)
```

Comment:
`cacheTime` is ambiguous, can we have a better/longer name that explains it better?